src/testcases/org/apache/poi/xssf/usermodel/helpers/XSSFColumnShifterTest.java

```diff
@@ -0,0 +1,384 @@
+package org.apache.poi.xssf.usermodel.helpers;
```

Comment:
Rename to TestXSSFColumnShifter.java
If you've written your unit test generically enough, you can have a single unit test, `BaseTestColumnShifter`, that can be used to test both the `HSSFColumnShifter` and `XSSFColumnShifter`. As a bonus, anything tested in the base unit test will guarantee that `HSSFColumnShifter` and `XSSFColumnShifter` behave the same.

ASL 2.0 header